app/styles/app.scss

```diff
@@ -392,6 +392,9 @@ td.path{
     }
 }
 /* password content */
+.no-user-select {
+    user-select: none !important;
```

Comment:
Please move this line into the 'blur' class so that it automatically applies the rules without adding another class.

Reply:
Done